### ISSUE TITLE
Not possible to submit Entry Notes inside a lightbox

### ISSUE THREAD

**Issue by rafaehlers** [GravityKit/GravityView#2181]
When running the Divi theme, this PHP warning occurs:

> Warning: Undefined array key "gravityview-fancybox" in /home/gagatoguru10 content/themes/Divi/inchides/builder/feature/JQueryBody.php on line 249

The issue also occurs when running twentytwenty, so it's not Divi-specific.

```
Uncaught ReferenceError: GVNotes is not defined
    at HTMLFormElement.add_note (entry-notes.js?ver=2.30.1:279:10)
    at HTMLFormElement.dispatch (jquery.js?ver=3.7.1:5145:27)
    at elemData.handle (jquery.js?ver=3.7.1:4949:28)
```

## Steps to reproduce

1. Create a new View for a form that has entries
2. Add a Link to Single Entry field in Multiple Entries screen
3. Enable "Open as lightbox" setting for that field
4. Add an Entry Notes field to Single Entry tab
5. Save the View
6. Open in frontend
7. Click the link to entry
8. In the lightbox, enter text in the entry note field and click "Add Note"
9. See error in JS console

![image](https://github.com/user-attachments/assets/0fe54aeb-92d1-4704-8dea-ea38a0878d70)

![image](https://github.com/user-attachments/assets/671db05f-9fdd-42d5-bd63-f5d7333653ab)

It can be tested here: https://debug.try.gravitykit.com/view/sample-data/

Tested on GV 2.30.1

**Comment by rafaehlers** [comment]
https://secure.helpscout.net/conversation/2745299672/58333

**Comment by rafaehlers** [comment]
I tested, and it works! I sent the build to the customer ✅

**Comment by rafaehlers** [comment]
Released in GV 2.31
